configure

```diff
@@ -29,7 +29,8 @@ if test ! -r $GAPPATH/sysinfo.gap ; then
 fi
 
 GAPPATH=$(cd ${GAPPATH} && pwd )
-
+SCRIPTFILE=$(readlink -f "$0")
+SCRIPTDIR=$(dirname "${SCRIPTFILE}")
```

Comment:
So, why all these complications just to get the dir? The script will only work correctly if the PWD is set to point at GAPJulid anyway, right? So you can just use `$PWD` ...